### ISSUE TITLE
Add possibility to select the specific repositories you want to execute the command in.

### ISSUE THREAD

**Issue by guustysebie** [introfog/rGit#15]
Lets say we have the following:
source folder:
    Repo 1:
    Repo 2:
    Repo 3:

And I would like to be able to only execute the command in "repo 1" and "repo 2"

**Comment by introfog** [comment]
Yeah, it is one of the most prioritized feature for the next release
